compact_str/src/repr/traits.rs

```diff
@@ -9,18 +9,22 @@ pub trait IntoRepr {
 }
 
 impl IntoRepr for f32 {
+    #[inline]
+    #[track_caller]
     fn into_repr(self) -> Repr {
         let mut buf = ryu::Buffer::new();
         let s = buf.format(self);
-        Repr::new(s)
+        Repr::new(s).unwrap()
```

Comment:
This should also be falliable since it's called in try_to_compact_string()

Reply:
You're right, fixed.